lib/skynet/logger.rb

```diff
@@ -16,15 +16,15 @@ def finish
     end
 
     def info(event, session_id: nil, info: {})
-      log(event, { session_id: session_id || @session_id, level: :info }.merge(info))
+      log(event, {session_id: session_id || @session_id, level: :info}.merge(info))
```

Comment:
@jeanmartin 
I wonder what is the idea behind storing session_id inside info if we also send it here `session: @session_id`:
```
RedisStream.default_logger.publish(event, {
        service: RedisStream.config.stream_key,
        session: @session_id,
        agent_session_id: @agent_session_id,
        info: info
})
```

I wanted to do just
```
  def info(event, info = {})
      log(event, {level: :info}.merge(info))
  end